### DISCUSSION TITLE
Can't Identify Model Differences

### DISCUSSION THREAD

**Discussion by DonkeySaddle** [Open-Systems-Pharmacology/Forum#1374] [Questions & Problems] [answered]
I cannot figure out why these two simulations are not completely identical. One is a simulation("po 100 mg SD caps fed") from the [traconazole](https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library/tree/master/Itraconazole) folder provided in the GIT library, the other I have built from scratch by using/importing the building blocks from the first model. However, when I run the two simulations the results are remarkably different. 

What am I missing? Is there an easy way to identify what's going on? 

Any help would be greatly appreciated. 

Thanks,

DS

[po 100 mg SD caps fed.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/9916473/po.100.mg.SD.caps.fed.zip)
[Testing_Itra.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/9916474/Testing_Itra.zip)

**Comment by PavelBal** [comment, marked as answer]
Comparing the simulations in MoBi reveals a difference in solubility:

![image](https://user-images.githubusercontent.com/25903040/199498390-67089066-b7e4-4235-8bcc-4421beac51a6.png)

**Reply by DonkeySaddle** [reply]
This is great, thanks so much! How were you able to compare the compounds like this?

**Reply by PavelBal** [reply]
Load both simulations in MoBi, multi-select them (holding the "CTRL" key and selecting them with the left-mouse-click), and select "Compare simulations..."

![image](https://user-images.githubusercontent.com/25903040/199589142-96ad7be7-9658-41a3-bf4d-94873eee678c.png)

**Comment by DonkeySaddle** [comment]
🤦 should have figured this out! Anyways, much thanks.
